waffle-compiler/src/findInputs.ts

```diff
@@ -12,8 +12,12 @@ export function findInputs(sourcePath: string) {
       const filePath = path.join(dir, file);
       if (isDirectory(filePath)) {
         stack.push(filePath);
-      } else if (file.endsWith('.sol') || file.endsWith('.vy')) {
-        inputFiles.push(filePath);
+      } else {
+        for (const extension of extensions) {
+          if (file.endsWith(extension)) {
+            inputFiles.push(filePath);
+          }
+        }
```

Comment:
This doesn't really solve the problem, while `.sol` files and `.vy` files are still mixed. Decision should be made based on configuration. #compilationtype